zkevm-circuits/src/witness/rw.rs

```diff
@@ -246,35 +247,49 @@ pub enum Rw {
 
 /// Rw table row assignment
 #[derive(Default, Clone, Copy, Debug)]
-pub struct RwRow<F> {
+pub struct RwRow<F, T> {
     pub(crate) rw_counter: F,
     pub(crate) is_write: F,
     pub(crate) tag: F,
     pub(crate) id: F,
     pub(crate) address: F,
     pub(crate) field_tag: F,
+    #[deprecated]
     pub(crate) storage_key: F,
+    pub(crate) storage_key_word: T,
+    #[deprecated]
     pub(crate) value: F,
+    pub(crate) value_word: T,
+    #[deprecated]
     pub(crate) value_prev: F,
+    pub(crate) value_prev_word: T,
     pub(crate) aux1: F,
+    #[deprecated]
     pub(crate) aux2: F,
+    pub(crate) aux2_word: T,
 }
 
-impl<F: Field> RwRow<F> {
-    pub(crate) fn values(&self) -> [F; 11] {
+// TODO figure out a better trait bound on T to support function word lo()/hi()
+impl<F: Field, T: ToVec<F> + Clone> RwRow<F, T> {
+    pub(crate) fn values(&self) -> [F; 15] {
         [
-            self.rw_counter,
-            self.is_write,
-            self.tag,
-            self.id,
-            self.address,
-            self.field_tag,
-            self.storage_key,
-            self.value,
-            self.value_prev,
-            self.aux1,
-            self.aux2,
+            vec![
+                self.rw_counter,
+                self.is_write,
+                self.tag,
+                self.id,
+                self.address,
+                self.field_tag,
+            ],
+            self.storage_key_word.clone().to_vec(),
```

Comment:
Because `RwRow`  can't introduce `Word<F>` that's why here need to use vector way. Also see explanation here https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1441/files#r1211032980

Reply:
Fixed this issue mention in comment :) https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1441#discussion_r1211062212